sdk/monitor/Azure.Monitor.OpenTelemetry.Exporter/CHANGELOG.md

```diff
@@ -4,6 +4,10 @@
 
 ### Features Added
 
+* Added support for "service.version" in Resource attributes.
+  This is mapped as Application Version in Application Insights.
```

Comment:
nit suggestion: It'd be nice if this makes it clear that this will be applied to all 3 (or 2) signals.